syncer/filter.go

```diff
@@ -112,13 +55,50 @@ func (s *Syncer) skipDMLEvent(table *filter.Table, eventType replication.EventTy
 	case replication.DELETE_ROWS_EVENTv0, replication.DELETE_ROWS_EVENTv1, replication.DELETE_ROWS_EVENTv2:
 		et = bf.DeleteEvent
 	default:
-		return false, terror.ErrSyncerUnitInvalidReplicaEvent.Generate(eventType)
+		filtered, err := s.filterOneEvent(table, bf.NullEvent, "")
+		if err != nil {
+			return false, err
```

Comment:
Maybe should return `terror.ErrSyncerUnitInvalidReplicaEvent.Generate(eventType)`?

Reply:
`needSkip, err := s.skipOneEvent(table, bf.NullEvent, "")` why we need check NullEvent here? maybe just return `terror.ErrSyncerUnitInvalidReplicaEvent.Generate(eventType)?` is enouth

Reply:
Because if the `table` can be skipped by `baList`, we needn't return error.

But it is indeed not good by call `skipOneEvent`. Let me modify it.

I also think return `terror.ErrSyncerUnitInvalidReplicaEvent.Generate(eventType)` is right.